src/coffee/stockimport.coffee

```diff
@@ -371,8 +371,18 @@ class StockImport
             .fetch()
             .then (result) =>
               @_updateInventory(entry, result.body, tryCount + 1)
+            .catch (err) =>
+              if (err.statusCode == 404)
+                debug "Stock not existing anymore (probably was deleted), create a new stock"
+                @client.inventoryEntries.create(entry)
+              else
```

Comment:
Hello @lojzatran ... this `.catch(err)` will be called also when `_updateInventory()` raise some error so the debug message can be a little bit misleading.

Reply:
https://github.com/sphereio/sphere-stock-import/pull/69/commits/f0227ce01505b32c5565cc6d784389281b01533d